package.json

```diff
@@ -22,7 +22,7 @@
 		"node": ">= 18"
 	},
 	"dependencies": {
-		"faucet-pipeline-core": "^2.0.0",
+		"faucet-pipeline-core": "^3.0.0",
```

Comment:
Looks like we don't actually _need_ v3; cf. https://github.com/faucet-pipeline/faucet-pipeline-js/pull/233/commits/c880cac6f9bb8186c9b7965cff4f49a02bbded24 (pending discussion in faucet-core, of course):

> note that we don't actually _need_ faucet-core v3 though because `await` doesn't care whether there's a promise or not

Reply:
Let's discuss this in core :+1:

Reply:
As we agreed on, I will adjust this

Reply:
Done :heavy_check_mark: